qiita_db/support_files/patches/python_patches/14.py

```diff
@@ -0,0 +1,15 @@
+from qiita_db.sql_connection import SQLConnectionHandler
+from os.path import basename
+
+# This changes all analysis files to be relative path instead of absolute
+
+conn_handler = SQLConnectionHandler()
+
+filepaths = conn_handler.execute_fetchall(
+    'SELECT f.* from qiita.filepath f JOIN qiita.analysis_filepath afp ON '
+    'f.filepath_id = afp.filepath_id')
+for filepath in filepaths:
+    filename = basename(filepath['filepath'])
```

Comment:
Just to verify: these files are still relative to a unique directory associated with a study or a meta-analysis, correct?